Docs: Document the common ring section (#4664)
grafana/loki: docs/sources/configuration/_index.md

# When true, the ingester, compactor and query_scheduler ring tokens will be saved to files in the path_prefix directory
# Loki will error if you set this to true and path_prefix is empty.
[persist_tokens: <boolean>: default = false]

# A common ring config to be used by all Loki rings.

> # If a common ring is given, its values are used as fallback to non-set ring values. For instance,

Reviewer: ```suggestion
# If a common ring is given, its values are used to define any undefined ring values. For instance,
```
